config/initializers/engine.rb

```diff
@@ -0,0 +1,7 @@
+module Hyrax
```

Comment:
It looks like setting it in application.rb works as well, I think that would be another way